tests/handlers/events-query.spec.ts

```diff
@@ -88,27 +86,12 @@ export function testEventsQueryHandler(): void {
       expect(reply.entries).to.not.exist;
     });
 
-    it('returns 400 if no filters are provided', async () => {
```

Comment:
Unrelated to this PR, but didn't like the duality between an undefined filter and empty array, so made filters required and allowed an empty array in the message descriptor.

This is the test I noticed the discrepancy with, so the test is irrelevant now.